modules/utxo-lib/src/bitgo/transaction.ts

```diff
@@ -82,6 +83,61 @@ export function setTransactionBuilderDefaults<TNumber extends number | bigint>(
   }
 }
 
+export function setPsbtDefaults(
+  psbt: UtxoPsbt<UtxoTransaction<bigint>>,
+  network: Network,
+  { version = getDefaultTransactionVersion(network) }: { version?: number } = {}
+): void {
+  switch (getMainnet(network)) {
+    case networks.bitcoincash:
+    case networks.bitcoinsv:
+    case networks.bitcoingold:
+      if (version !== 2) {
+        throw new Error(`invalid version`);
+      }
+      break;
+    case networks.zcash:
+      throw new Error('Zcash psbt not implemented');
+    default:
+      if (version !== 1) {
+        throw new Error(`invalid version`);
+      }
+  }
+  // FIXME: Always call this, because there's a bug in the upstream PSBT that
+  // defaults transactions to v2.
+  psbt.setVersion(version);
```

Comment:
Is it safe to be setting version to `undefined` in the case that it's not specified in the params? Or should we default `version = 2` in case it's not provided?

Reply:
on line 89, `getDefaultTransactionVersion`, and we throw in the switch if it's not set to expected values.

I think there is a bug here that we say that we don't support zcash, but we do. Will have to look into that.